web/src/reducers/dataReducerForHistoryFeature.js

```diff
@@ -183,10 +183,11 @@ const reducer = (state = initialDataState, action) => {
         isLoadingHistories: false,
         histories: {
           ...state.histories,
-          [action.zoneId]: action.payload.map((datapoint) => ({
+          [action.zoneId]: action.payload.zoneStates.map((datapoint) => ({
```

Comment:
Dunno if I missed this on the `app-backend`, but did we change the response? :)

Reply:
Also, should the mockserver be updated to match this?

Reply:
The mockserver data also serves the zonesStates for history